src/main/java/pro/paullezin/jwtdemo/config/WebSecurityConfig.java

```diff
@@ -52,21 +52,21 @@ public UserDetailsService userDetailsService() {
     @Autowired
     public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
         auth.userDetailsService(userDetailsService())
-                .passwordEncoder(PASSWORD_ENCODER);
+                .passwordEncoder(passwordEncoder);
     }
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        CustomAuthenticationFilter customAuthenticationFilter = new CustomAuthenticationFilter(authenticationManagerBean(), jwtPropertyProvider);
-        customAuthenticationFilter.setFilterProcessesUrl("/api/login");
+        JwtAuthenticationFilter jwtAuthenticationFilter = new JwtAuthenticationFilter(authenticationManagerBean(), jwtPropertyProvider);
+        jwtAuthenticationFilter.setFilterProcessesUrl("/api/login");
```

Comment:
Do not create instance manually: Spring can do it itself. You need just to Autowire this bean